### PR TITLE
[DOC] Added R to the list of languages with "high-level API" support in the…

### PR DESCRIPTION
**Pull request by enragedginger**
… main README.

### CODE REVIEW

**Comment by enragedginger** [comment]
I just added another commit which makes some spelling / grammar updates to the quick-start MD file.

File: docs/quick-start.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ scalaVersion := "{{site.SCALA_VERSION}}"
 libraryDependencies += "org.apache.spark" %% "spark-core" % "{{site.SPARK_VERSION}}"
 {% endhighlight %}
 
-For sbt to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
+For SBT to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
```
Comment (srowen) [inline docs/quick-start.md]: sbt is lower-case. jar seems equally upper and lower case, so, fine to change but on the whole I tend to leave things like that. I think the other changes are OK but not sure I'd bother with things that aren't obviously wrong, like an Oxford comma.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Apache Spark
 
 Spark is a fast and general cluster computing system for Big Data. It provides
-high-level APIs in Scala, Java, and Python, and an optimized engine that
+high-level APIs in Scala, Java, Python, and R, and an optimized engine that
```
Comment (srowen) [inline README.md]: I'm not clear whether R is advertised as stable enough for general use?

**Comment by srowen** [comment]
Although this is probably OK without a JIRA, that's exceptional. Please read https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark first. In particular this isn't tagged as `[DOCS]` for example. Minor stuff but I don't want to let people to skip this.

File: README.md (hunk 2 above)
Comment (enragedginger) [inline README.md]: R is listed in the first paragraph on [this page](http://spark.apache.org/docs/latest/index.html), so I figured it should be in the README.md as well.

File: docs/quick-start.md (hunk 1 above)
Comment (enragedginger) [inline docs/quick-start.md]: I've switched all usages of "SBT" back to "sbt". At one point in time, SBT stood for "Simple Build Tool," but I can't find any remaining evidence of that on their site, so I'm guessing they've moved away from that.

File: docs/quick-start.md [hunk 3]
```diff
@@ -217,13 +217,13 @@ a cluster, as described in the [programming guide](programming-guide.html#initia
...
-Now say we wanted to write a self-contained application using the Spark API. We will walk through a
-simple application in both Scala (with SBT), Java (with Maven), and Python.
+Suppose we wish to write a self-contained application using the Spark API. We will walk through a
+simple application in Scala (with sbt), Java (with Maven), and Python.
 
 <div class="codetabs">
 <div data-lang="scala" markdown="1">
 
-We'll create a very simple Spark application in Scala. So simple, in fact, that it's
+We'll create a very simple Spark application in Scala--so simple, in fact, that it's
```
Comment (srowen) [inline docs/quick-start.md]: Fair enough about R. This LGTM. I would have put spaces around this dash but it's too small to bother with.

**Pull request closed without merging**